### PR TITLE
Add service account definition

### PR DESCRIPTION
**Pull request by kevinrizza**
-Create sa.yaml to conform to operator-framework format

### CODE REVIEW

File: deploy/sa.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: default
```
Comment (aravindhp) [inline deploy/sa.yaml]: Please call it `marketplace-operator-serviceaccount.` This change should also be reflected in rbac.yaml. And please run the deployment to ensure that everything works as before.

File: deploy/rbac.yaml [hunk 2]
```diff
@@ -45,7 +45,7 @@ metadata:
   name: default-account-marketplace-operator
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: marketplace-operator-serviceaccount
   namespace: marketplace
```
Comment (tkashem) [inline deploy/rbac.yaml]: You would also need to change change the deployment spec to use this service account in `operator.yaml` file, I believe.
```
   spec:
      serviceAccountName: marketplace-operator-serviceaccount
```

**Comment by SamiSousa** [comment]
For #19, should be setting `namespace: marketplace` in `operator.yaml` so that marketplace operator's namespace matches the namespace for CatalogSourceConfig and OperatorSource.

File: deploy/sa.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: marketplace-operator-serviceaccount
+  namespace: marketplace
```
[the remark this answers is not in the document]
Comment (SamiSousa) [reply deploy/sa.yaml]: I also recommend setting `namespace: marketplace` for the example CRs as well.

**Pull request closed without merging**